scipy/stats/_binomtest.py

```diff
@@ -300,7 +300,8 @@ def binomtest(k, n, p=0.5, alternative='two-sided'):
     if alternative not in ('two-sided', 'less', 'greater'):
         raise ValueError("alternative not recognized; \n"
                          "must be 'two-sided', 'less' or 'greater'")
-
+    
+    a_fn = lambda x1:binom.pmf(x1,n,p)
```

Comment:
```suggestion
    a_fn = lambda x1: binom.pmf(x1, n, p)
```
Does this need to be created up here or can it be done closer to where it's needed?
For instance, if it's going to be a `lambda` function, it could be passed into `binary_search_for_binom_tst` function directly.
If not, PEP8 suggests using `def` to create a named function. In that case, please use a more explicit name like `binom_pmf_np`.

Reply:
Moved inside the function call itself as suggested. PEP8 is okay with this.